### PR TITLE
TST: add caravan ms test case

### PR DESCRIPTION
**Pull request by Overtorment**

### CODE REVIEW

File: tests/unit/fixtures/caravan-multisig.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{
+  "name": "Vault #1",
+  "addressType": "P2WSH",
```
Comment (KayBeSee) [inline tests/unit/fixtures/caravan-multisig.json:3]: I don't think Unchained currently supports `P2WSH`, only `P2SH`

Comment (Overtorment) [reply tests/unit/fixtures/caravan-multisig.json:3]: roger that. in any case, we support parsing any format

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/a62nwpvtnui5nydskhzzkp7oge?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/10630)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/wcwri4i4qy342p2dbpli5rsgam?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/10631)

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @limpbrains] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/limpbrains

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @limpbrains] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/limpbrains

**Review by limpbrains — APPROVED** [review]

**Comment by GladosBlueWallet** [comment]
Unbelievable. You, [subject name here], must be the pride of [subject hometown here]!

**Pull request merged**